helm/charts/k8s-image-availability-exporter/README.md

```diff
@@ -40,8 +40,8 @@ The following tables list the configurable parameters of the k8s-image-availabil
 | Parameter | Description | Default |
 | ----- | ----------- | ------ |
 | `k8sImageAvailabilityExporter.image.pullPolicy` | Image pull policy to use for the k8s-image-availability-exporter deployment | `IfNotPresent` |
-| `k8sImageAvailabilityExporter.image.repository` | Repository to use for the k8s-image-availability-exporter deployment | `flant/k8s-image-availability-exporter` |
-| `k8sImageAvailabilityExporter.image.tag` | Tag to use for the k8s-image-availability-exporter deployment | `v0.1.13` |
+| `k8sImageAvailabilityExporter.image.repository` | Repository to use for the k8s-image-availability-exporter deployment | `ghcr.io/Webfleet-Solutions/k8s-image-availability-exporter` |
```

Comment:
mmh, our common abbreviation for group is either "wfs" or "webfleet", even for things we published outside. On the other hand the chosen identifier is easiear to assimilate with our org in GitHub. So, might be okay to leave it like that.

Reply:
I agree, but unfourtunately, this is out of our hands, as the Github Organization is named this way. This will always be the identifier of ghcr packages.